.gitignore

```diff
@@ -10,10 +10,17 @@ tmp/
 /build
 node_modules/
 static/js/lib/stm_web.min.js
+package-lock.json
```

Comment:
I think we're now keeping `package-lock.json` in the repo for people not using `yarn`.